libs/utils/src/ostream.cpp

```diff
@@ -87,142 +94,116 @@ const char* ostream::getFormat(ostream::type t) const noexcept {
     }
 }
 
-void ostream::growBufferIfNeeded(size_t s) noexcept {
-    Buffer& buf = getBuffer();
-    if (UTILS_UNLIKELY(buf.size < s)) {
-        size_t used = buf.curr - buf.buffer;
-        size_t newCapacity = std::max(size_t(32), used + (s * 3 + 1) / 2); // 32 bytes minimum
-        buf.reserve(newCapacity);
-        assert(buf.size >= s);
+UTILS_NOINLINE
+ostream& ostream::print(const char* format, ...) noexcept {
```

Comment:
Do we have any need for more than 1 argument? Or are you future-proofing?

Reply:
actually it's not about have one argument, it's about not knowing what type it is! I didn't want to use templates and generate tons of code.